src/main/resources/frontend/src/views/AdminView/SubmissionsView.vue

```diff
@@ -20,25 +20,36 @@ import {nameFromNetId} from "@/utils/utils";
 
 const selectedSubmission = ref<Submission | null>(null);
 const selectedStudent = ref<User | null>(null);
+const DEFAULT_SUBMISSIONS_TO_LOAD = 25;
+let allSubmissionsLoaded = false;
 
 onMounted(async () => {
-  const submissionsData = await submissionsLatestGet();
-  submissionsData.sort((a, b) => b.timestamp.localeCompare(a.timestamp)) // Sort by timestamp descending
+  const submissionsData = await submissionsLatestGet(DEFAULT_SUBMISSIONS_TO_LOAD);
+  //submissionsData.sort((a, b) => b.timestamp.localeCompare(a.timestamp)) // Sort by timestamp descending
```

Comment:
It seems this line has been removed, and you are relying on the server returning sorted results to you.

## Trusting Results to be Pre Sorted
Per the comment above, that isn't strictly defined which makes it an unsafe assumption for now. Breaking the fourth wall to go look at the server code from the client, it seems that the SQLSubmissionDAO uses an `ORDER BY` clause which will do the behavior, but it seems that the MemorySubmissionDAO **does not** manage the order of the returned results.

It's not safe to trust that the results will come to you properly until it's defined in the API. At that point, we can point fingers at the server if things break, but honestly, the client should be responsible for sorting the results because they'll usually want diverse kinds of sorts.

## Leaving the comment behind
If after discussing all the reasons, it's decided that this line isn't needed, the line should be removed entirely. We can always retrieve the code again from the `git` file history. There's no need to leave commented out code in the file in case you need it again. Besides, we would dock our students for this behavior (I hope)!

Reply:
The comment should be removed, and I will do so promptly. It was left there while I was making sure that it was still sorted after refactoring some things. 

The commented code is not needed, AND we are not trusting the results to be pre-sorted. The table automatically sorts the data descending based on the timestamp column.